src/Credentials/InsecureCredentials.php

```diff
@@ -67,4 +67,15 @@ public function getLastReceivedToken()
     {
         return $this->token;
     }
+
+    /**
+     * Get the client ID. In this case, it returns an empty string.
```

Comment:
Small nit, let's stay consistent in our use of `client name` across these docs.

Reply:
done, PTAL/